internal/cloud/awscloud/awscloud.go

```diff
@@ -290,7 +290,7 @@ func (a *AWS) Register(name, bucket, key string, shareWith []string, rpmArch str
 			Tags: []*ec2.Tag{
 				{
 					Key:   aws.String("Name"),
-					Value: aws.String(name),
+					Value: aws.String(key),
```

Comment:
Hm so this does kinda break on-prem behaviour, since the key set in the target file now becomes the ami name (instead of the 'name' set in the target file). Maybe having a 2nd tag makes sense after all.